\documentclass[11pt]{article}
\renewcommand \thesection{\Alph{section}}
\makeatletter
\renewcommand\section{\@startsection {section}{1}{\z@}%
{-1.5ex \@plus -1ex \@minus -.2ex}%
{1.5ex \@plus.2ex}%
{\color{dukeblue}\sffamily\Large\bfseries}}
\makeatother

% Packages
\usepackage[letterpaper,hmargin=0.5in,vmargin=0.5in]{geometry}
\usepackage{mathpazo}
\usepackage{amsmath}
\usepackage[dvips]{graphicx}
\usepackage{floatflt}
\usepackage{color}
\usepackage{boxedminipage}
\usepackage{url}
\usepackage{soul}
\usepackage[numbers,super,comma,sort&compress]{natbib}
\usepackage{multirow}
\usepackage{xspace}
%\usepackage[scaled]{helvet}
\usepackage{helvet}

\usepackage[rightcaption]{sidecap}
\usepackage{calc}

%Pretty
\definecolor{dukeblue}{rgb}{0.000,0.102,0.341} % hex triple is 001A57

%\definecolor{light-gray}{gray}{0.3}
%\definecolor{darkgray}{gray}{0.15}
%\definecolor{duke}{rgb}{0,0,0.611}
\definecolor{ourblue}{rgb}{0.00,0.15,0.55} % notably brighter than dukeblue (but all text must be black, so ditching this)

 \renewcommand{\familydefault}{\sfdefault}


\renewcommand{\labelitemi}{\color{dukeblue}$\bullet$}

% Bibliography
%\bibpunct[,]{~[}{]}{,}{n}{}{}
\bibliographystyle{alexunsrtabbrvnat}
% \setlength\bibsep{1.8pt}
\renewcommand{\refname}{Literature~Cited}

\newcommand{\captionfonts}{\sf \footnotesize \color{dukeblue}}
\makeatletter % Allow the use of @ in command names
\long\def\@makecaption#1#2{%
\vskip\abovecaptionskip
	\sbox\@tempboxa{{\captionfonts #1. #2}}%
	\ifdim \wd\@tempboxa >\hsize
		{\captionfonts \textbf{#1.} #2\par}
	\else
		\hbox to\hsize{\hfil\box\@tempboxa\hfil}%
	\fi
	\vskip\belowcaptionskip}
\makeatother % Cancel the effect of \makeatletter
\renewcommand{\figurename}{Figure}

%\def\mycaptionsize{\footnotesize}
\def\mycaptionsize{\small}

\def\mycodesize{\footnotesize}
\def\myeqnsize{\small}

% Section Heading
\newcounter{sheading}[section]
\setcounter{sheading}{0}
\newcounter{ssheading}[sheading]
\setcounter{ssheading}{0}
\newcounter{sssheading}[ssheading]
\setcounter{sssheading}{0}

\def\sheading#1{\medskip\noindent\refstepcounter{sheading}{\sffamily \color{dukeblue} \large \textbf{\Alph{section}.\arabic{sheading}\hspace{1em}#1}\medskip\setcounter{ssheading}{0}}\\}


%\def\ssheading#1{\noindent\refstepcounter{ssheading}{\sffamily \color{dukeblue} \bfseries \textsl {\Alph{section}.\arabic{sheading}.\arabic{ssheading}\hspace{1em}#1}\hspace{.7em}}}

\def\ssheading#1{\noindent\refstepcounter{ssheading}{\sffamily \color{dukeblue} \bfseries \textsl {#1}\hspace{.7em}}}

%\linespread{1.00}
\def\degree{$^\circ$}
\def\R{\mathbb{R}}
\def\CR{\hspace{0pt}} % ``invisible'' space for line break
\newcommand\myrefa[2]{\ref{#1}.\ref{#2}}
\newcommand\myrefb[3]{\ref{#1}.\ref{#2}.\ref{#3}}
\newcommand\myrefc[4]{\ref{#1}.\ref{#2}.\ref{#3}.\ref{#4}}

% Biological Abbreviations
\newcommand\dros{{\itshape Drosophila}\xspace}
\newcommand\dmel{{\itshape D.~melanogaster}\xspace}
\newcommand\scer{{\itshape S.~cerevisiae}\xspace}
\newcommand\saccer{{\itshape Saccharomyces cerevisiae}\xspace}
\newcommand\xenopus{{\itshape Xenopus}\xspace}
\newcommand\invitro{{\itshape in~vitro}\xspace}
\newcommand\invivo{{\itshape in~vivo}\xspace}
\newcommand\ten[1]{$\times$10$^{#1}$}
\newcommand\panc{Panc\dash1\xspace}
\newcommand\orcmt{\emph{orc1\dash 161}\xspace}

% Other Abbreviations
\newcommand\eg{\emph{e.g.}\xspace}
\newcommand\dash{\nobreakdash-\hspace{0pt}}

\def\be{\begin{enumerate}} % Begin Enumerate
\def\ee{\end{enumerate}} % End Enumerate
\def\en{\item} % Entry (item)
\def\bi{\begin{itemize}} % Begin Itemize
\def\ei{\end{itemize}} % End Itemize
\def\bv{\begin{verbatim}} % Begin Verbatim
\def\ev{\end{verbatim}} % End Verbatim
\begin{document}
\pagestyle{empty}
\addtolength{\itemsep}{-5mm}
\addtolength{\parskip}{1.5mm}
\addtolength{\abovecaptionskip}{-10mm}
%\input{r35_1_background_2.tex}
%\input{r35_2_progress.tex}
%\input{r35_3_future.tex}
\section*{Abstract}
Our research is focused on elucidating the mechanisms by which the local chromatin environment influences DNA-templated processes including DNA replication, transcription and DNA repair.  While considerable progress has been made in our understanding of the mechanisms that direct DNA replication \invitro, we know very little about how start sites of DNA replication (origins) are selected and regulated in the context of the chromosome.  The genomic approaches that my research group have pioneered have provided new insights into the mechanisms by which the local chromatin state and structure (nucleosome and transcription factor occupancy) influences key steps in regulating the DNA replication program in multiple species including \scer and \dros.  We have recently developed a novel approach to `footprint' a eukaryotic genome -- simultaneously revealing genome-wide occupancy of DNA for both nucleosomes and smaller DNA binding factors (\eg initiation and transcription factors).  Unlike biochemical reconstitution experiments utilizing one or two defined DNA templates, we are able to comprehensively view the cell cycle regulated cascade of chromatin changes that occur surrounding each origin of replication in the yeast genome.  Our future research will focus on identifying and characterizing the chromatin mediated events required for initiation of DNA replication following helicase loading.  We will also investigate how chromatin structure is re-established throughout the genome to preserve epigenetic integrity following passage of the DNA replication fork.  DNA replication is also a potent source of double-stranded breaks (DSB) which, if not repaired, may lead to genomic instability.  We are uniquely positioned to identify and understand the dynamics of chromatin structure following the induction of site-specific DSBs and their subsequent repair by homologous recombination or non-homologous end joining. Finally, in collaboration with the Hartemink laboratory (Duke, CS) we are using synchronous populations of yeast proceeding through the cell cycle to develop robust statistical approaches that will enable us to model cell cycle-dependent changes in gene expression from chromatin occupancy data. 
\end{document}